source/core/ut_suite_manager.pkb

```diff
@@ -592,6 +592,10 @@ create or replace package body ut_suite_manager is
       l_need_all_objects_scan := false;
     end if;
 
+    for i in 1 .. a_schema_names.count loop
+      refresh_cache(a_schema_names(i));
+    end loop;
+
     execute immediate 'select c.object_owner, c.object_name
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: An illegal character with code point 10 was found in this literal. ([plsql:LiteralsNonPrintableCharactersCheck](https://sonarcloud.io/organizations/utplsql/rules#rule_key=plsql%3ALiteralsNonPrintableCharactersCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=utPLSQL&pullRequest=799&issues=AWdNdTLbiCTqmtbHBs04&open=AWdNdTLbiCTqmtbHBs04)

[ISSUE_KEY]: AWdNdTLbiCTqmtbHBs04